frontend/components/users/profile/ShareCalendarModal.tsx

```
@@ -0,0 +1,172 @@
+                  <label className="block text-sm font-medium text-gray-700 mb-2">Share Link</label>
+                  <div className="bg-gray-100 rounded-md p-3 font-mono text-xs text-gray-800 break-all">
+                    {getShareLink()}
+                  </div>
+                </div>
+
+                {/* Action Buttons */}
+                <div className="flex gap-3">
+                  <button
+                    onClick={handleClose}
+                    className="flex-1 px-4 py-2 text-sm font-medium text-gray-700 bg-white border border-gray-300 rounded-md hover:bg-gray-50 transition-colors"
+                  >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)